composer.json

```diff
@@ -394,6 +394,9 @@
             },
             "drupal/responsive_favicons": {
                 "Issue #3376766: Incorrect path if Drupal is installed in subfolder": "https://git.drupalcode.org/project/responsive_favicons/-/merge_requests/6.diff"
+            },
+          "drupal/editor_advanced_link": {
+                "Issue #3423208: Integrate a functionality of the CKEditor Bootstrap Buttons module": "https://git.drupalcode.org/project/editor_advanced_link/-/merge_requests/19.diff"
```

Comment:
MR is in DRAFT, we can't let it in @aleevas @rollki 
Could you make it Ready?

Reply:
Hi @podarok 
Of course, thank you!

Reply:
Also, I think we need to use a static patch here and not a diff merge request, what do you think?

Reply:
cc @aleevas